src/pandas_profiling/profile_report.py

```diff
@@ -172,6 +175,12 @@ def description_set(self):
             )
         return self._description_set
 
+    @property
+    def infer_dtypes(self):
```

Comment:
This isn't used (and not needed, see line 81)

Reply:
Yes, not needed.
We can remove this function as well, including the lines which has `infer_dtypes` word